client/paperflow-web-v1/src/pages/KeywordNetwork/index.js

```diff
@@ -25,21 +43,59 @@ const App = () => {
           });
         select(svgRef.current).call(container);
       }
-    }, 1000);
+    }, 10);
     return () => clearTimeout(timer);
   }, []);
 
+  const [networkData, setNetworkData] = useState({
+    nodes: [],
+    links: [],
+  });
+
+  let i;
+  let j;
```

Comment:
is there any reason to put `i`, `j` here? if there's no reason, remove line 55, 56